timeless/restaurants/table_shapes/forms.py

```diff
@@ -9,17 +9,8 @@
 class TableShapeForm(forms.ModelForm):
     """Base form for table shape"""
 
-    picture = FileField(
-        "Table Shape Image",
-        validators=[FileRequired(), FileAllowed(IMAGES, "Images only!")])
-
```

Comment:
@vladarefiev I don't really understand, why did you remove the ``picture`` field and what do we have instead of it? :D

Reply:
@amihaiemil I have no idea why it's was overridden,. it was mistake ;)